spec/factories/eras.rb

```diff
@@ -1,8 +1,9 @@
 FactoryBot.define do
+  date = Faker::Date.between(from: 10.years.ago, to: 1.year.ago)
   factory :era do
     name { Faker::Hipster.sentence }
-    start_date { '2020-12-19 15:37:22' }
-    end_date { '2020-12-19 15:37:22' }
+    start_date { date }
+    end_date { Faker::Date.between(from: date, to: Date.today) }
     color { Faker::Color.hex_color }
```

Comment:
I like this change!

Reply:
Nice! Does `date` on line 5 give a random date?

Reply:
It gives a random date between 10 years ago and 1 year ago. Defined on line 2!

Reply:
Nice, didn't know you could do that with dates.